crates/biome_css_parser/src/lexer/mod.rs

```diff
@@ -944,7 +946,7 @@ impl<'src> CssLexer<'src> {
 
                 COMMENT
             }
-            _ => self.consume_unexpected_character(),
+            _ => self.consume_byte(T![/]),
```

Comment:
Could you please explain why we need to consume `T![/]`?

Reply:
in css ratio 1 / 3